.swiftlint.yml

```diff
@@ -0,0 +1,41 @@
+disabled_rules:
+- trailing_whitespace
+- line_length
```

Comment:
Why disabled? It's used below?

Reply:
My mistake. I was testing and forgot to revert. Will correct.